python/engine/test/WrongMethodMeasure/measure.py

```diff
@@ -0,0 +1,30 @@
+import typing
+
+import openstudio
+
+
+class WrongMethodMeasure(openstudio.measure.ModelMeasure):
+    def name(self):
+        return "Wrong Method Measure"
+
+    def modeler_description(self):
+        return "The arguments method calls a non existing method on the model passed as argument"
+
+    def arguments(self, model: typing.Optional[openstudio.model.Model] = None):
+        model.nonExistingMethod()
+        args = openstudio.measure.OSArgumentVector()
```

Comment:
Added a test for both python and ruby that calls a wrong method on model passed as argument in arguments()